### PR TITLE
arm64: dts: qcom: motorola-ocean: Add dsi panel entry

### PR DESCRIPTION
**Pull request by Ultracoolguy**
Feel free to edit/reword the commit.

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts [hunk 1]
```diff
@@ -89,14 +89,60 @@
 
 &msmgpio {
 	gpio-reserved-ranges = <96 4>;
+
+	pmx_mdss_default: pmx_mdss_default {
```
Comment (scarface-one) [inline arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:93]: indentation is funky
Use hardware tabs whenever possible

**Review by scarface-one — COMMENTED** [review]
Other than indentation this looks fine to me, but Junak should probably review since he made the generic DSI driver

File: arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts (hunk 1 above)
Comment (Ultracoolguy) [reply arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:93]: Gotcha. Fixed.

**Comment by Ultracoolguy** [comment]
Closing this PR since junak already put the entry in the dts.

**Pull request closed without merging**